src/dpp/message.cpp

```diff
@@ -882,6 +882,38 @@ bool attachment::is_remix() const {
 	return flags & a_is_remix;
 }
 
+time_t attachment::get_expire_time() const {
+	size_t attr_position = url.find('?');
+	/*If no attributes were sent in url, we do not need to parse more*/
```

Comment:
Space the comments correctly!
`/* If ......... */`

Reply:
Done